pyscriptjs/src/pyodide.ts

```diff
@@ -97,6 +97,32 @@ export class PyodideRuntime extends Runtime {
         }
     }
 
+    /**
+     *
+     * @param path : the path in the filesystem
+     * @param fetch_path : the path to be fetched
+     *
+     * For a given resource with `fetch_path` as `http://dummy.com/hi.py`
+     * and the location of `path` in the FS to be `a/b/c/foo.py`
+     *
```

Comment:
Do you think it makes sense adding an example in the format: `await loadFromFile("http://dummy.com/hi.py", "a/b/c/foo.py")`?

Just wondering if a visual cue may be a good idea

Reply:
Thanks, added!